Sources/ClientRuntime/EventStream/MessageUnmarshallable.swift

```diff
@@ -5,25 +5,4 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-/// Unmarshals a `Message` into a event stream event.
```

Comment:
The `MessageUnmarshallable` protocol and the helper method are no longer needed since closures are used to marshal & unmarshal.